daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/processors/parsers/BlobLengthParsers.scala

```diff
@@ -32,15 +31,18 @@ trait ByteChunkWriter { self: PrimParser =>
    * which takes as parameters an array of bytes and the number of bytes in
    * that array that should be written. The chunk size is defined by the
    * blobChunkSizeInBytes tunable.
+   *
+   * @return the actual number of bits written
```

Comment:
I think this no longer returns number of bits written? Can this comment be removed?